class-sc-wp-updates-notifier.php

```diff
@@ -438,6 +471,45 @@ public function check_themes_against_notified( $themes_need_update ) {
 		}
 
 
+		/**
+		 * Prepare the message.
+		 *
+		 * @param array $updates Array of all of the updates to notifiy about.
+		 * @param array $markup_vars Array of the markup characters to use.
+		 *
+		 * @return string Message to be sent.
+		 */
+		public function prepare_message( $updates, $markup_vars ) {
+			$message = $markup_vars['i_start'] . __( 'Updates Available', 'wp-updates-notifier' ) . $markup_vars['i_end'] . $markup_vars['line_break'] . $markup_vars['b_start'] . esc_html( get_bloginfo() ) . $markup_vars['b_end'] . ' - ' . $markup_vars['link_start'] . esc_url( home_url() ) . $markup_vars['link_middle'] . esc_url( home_url() ) . $markup_vars['link_end'] . $markup_vars['line_break'];
```

Comment:
Translated text ("Updates Available") should be escaped because it's dynamic. The translation could introduce unwanted characters.

Reply:
Readability could be improved here by breaking up all of these long lines so that no horizontal scrolling is needed.